runtime/basilisk/src/xcm.rs

```diff
@@ -268,6 +273,9 @@ impl pallet_xcm::Config for Runtime {
 	type WeightInfo = weights::xcm::BasiliskWeight<Runtime>;
 	#[cfg(feature = "runtime-benchmarks")]
 	type ReachableDest = ReachableDest;
+	type AdminOrigin = SuperMajorityCouncilOrRoot;
```

Comment:
we probably dont want council to have this privilege

Reply:
SuperMajorityTechCommitteeOrRoot ?